meta-buildpacks/clojure/buildpack.toml

```diff
@@ -0,0 +1,32 @@
+api = "0.9"
+
+[buildpack]
+id = "heroku/clojure"
+version = "3.2.0"
+name = "Scala"
+homepage = "https://github.com/heroku/buildpacks-jvm"
+description = "Official Heroku buildpack for Clojure applications."
+keywords = ["clojure"]
```

Comment:
Updating the Docker Hub descriptions reminded me of this - we'll need to update the descriptions/keywords etc here to use the new style introduced in #603.